be/src/exec/vectorized/hash_joiner.cpp

```diff
@@ -66,6 +66,8 @@ Status HashJoiner::prepare_builder(RuntimeState* state, RuntimeProfile* runtime_
         runtime_profile->add_info_string("Predicates", _hash_join_node.sql_predicates);
     }
 
+    runtime_profile->add_info_string("DistributionMode", to_string(_hash_join_node.distribution_mode));
+    runtime_profile->add_info_string("JoinOp", to_string(_hash_join_node.join_op));
```

Comment:
has alread add one JoinType.
```
runtime_profile->add_info_string("JoinType", _get_join_type_str(_join_type));
```

Reply:
In pipeline-engine, `HashJoinNode::prepare` will not be executed.

Reply:
runtime_profile->add_info_string("JoinType", _get_join_type_str(_join_type));

should be removed ?

Reply:
I misunderstood, I will remove one of the duplicate code